snippets/swift/BreezSDKExamples/Sources/ClosedChannel.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  ClosedChannel.swift
+//
+//
+//  Created by ruben on 12/12/2023.
+//
+import Foundation
+
+import BreezSDK
+
+func prepareRedeemOnchainFunds(sdk: BlockingBreezServices, feeRate: UInt64) -> PrepareRedeemOnchainFundsResponse? {
+    // ANCHOR: prepare-redeem-onchain-funds
+    let satPerVbyte = feeRate
+    let redeemOnchainFundsRequest = try? sdk.prepareRedeemOnchainFunds(req: PrepareRedeemOnchainFundsRequest(toAddress: "bc1..", satPerVbyte: satPerVbyte))
```

Comment:
Think the lines are too long here and hard to read on small devices
```suggestion
    let req = PrepareRedeemOnchainFundsRequest(toAddress: "bc1..", satPerVbyte: satPerVbyte)
    let redeemOnchainFundsResponse = try? sdk.prepareRedeemOnchainFunds(req: req)
```

Reply:
Good point, I agree.